### ISSUE TITLE
Custom Layers and Loss Functions

### ISSUE THREAD

**Issue by patrickespake** [patrickespake/ai-mastery-12-months#67]
Create custom components for neural networks in both frameworks.
